src/gda/backend_gda.cpp

```diff
@@ -818,10 +848,16 @@ void GDABackend::modify_qps_init_to_rtr() {
   memset(&attr, 0, sizeof(struct ibv_qp_attr));
   attr.qp_state               = IBV_QPS_RTR;
   attr.path_mtu               = portinfo.active_mtu;
-  attr.max_dest_rd_atomic     = GDA_MAX_ATOMIC;
   attr.min_rnr_timer          = 12;
   attr.ah_attr.port_num       = port;
 
+  if (nic_type == GDA_NIC::IONIC) {
+    attr.max_dest_rd_atomic = 15;
+  }
+  else if (nic_type == GDA_NIC::BNXT || nic_type == GDA_NIC::MLX5) {
```

Comment:
```suggestion
  } else {
```
`attr.max_dest_rd_atomic = 1` seems to be the default value for the supported NIC types, only IONIC is different

Reply:
I kept it explicit so we wouldn't forget but this is pretty valid